src/translations/en.json

```diff
@@ -128,7 +148,7 @@
     "ledger_connect_unlock": "Connect and unlock your Ledger Device",
     "ledger_open_eth_app": "Open the Ethereum app on your device",
     "ledger_transaction_cancelled_error": "Transaction Cancelled",
-    "ledger_contract_transaction_data_error": "Failed to transfer token. Please make sure 'Contract data' is set to yes on your Ledger device.",
+    "ledger_contract_transaction_data_error": "Failed to sign transaction for contracts. Please make sure 'Contract data' is set to yes on your Ledger device.",
```

Comment:
This is OK in the Chinese translation already?

Reply:
Thanks for the reminder. I have updated the missed Chinese translation.